### PR TITLE
KairosDB queries in ZMON dashboard widgets

### PR DESCRIPTION
**Pull request by ePaul**
Based on some discussion in the zalando-internal chat #zmon-users (thanks @Jan-M), and own experiments. I also added the widget in the documentation to https://demo.zmon.io/#/dashboards/view/1.

### CODE REVIEW

File: docs/user/dashboards.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ ZMON's customizable dashboards enable you to configure widgets and choose which
         dashboard has. An example of a valid widget configuration is the
         following:
 
-        .. code-block:: yaml
+        .. code-block:: json
```
Comment (ePaul) [inline docs/user/dashboards.rst:34]: This fixes a warning by sphinx. There are still many others, I gave up.

**Comment by Jan-M** [comment]
:+1:

**Comment by Jan-M** [comment]
Thanks for writing this down!

**Comment by ePaul** [comment]
:+1:

**Pull request merged**